jwst/resample/resample_spec_step.py

```diff
@@ -23,15 +23,21 @@ class ResampleSpecStep(ResampleStep):
     """
 
     def process(self, input):
-        input = datamodels.open(input)
-
+        input_new = datamodels.open(input)
+        input_new.meta.bunit_err = None
```

Comment:
This can't work, as `input` may not have `meta.bunit_err`. I.e. if it is a MultiSlitModel or ModelContainer.